Source/Target.swift

```diff
@@ -49,14 +49,23 @@ public enum StructTarget: TargetType {
 
 /// Represents an HTTP method.
 public enum Method: String {
-    case GET, POST, PUT, DELETE, OPTIONS, HEAD, PATCH, TRACE, CONNECT
+
+    case options = "OPTIONS"
```

Comment:
Hmm, didn't really check the code so forgive me, but is there a reason we cannot use typealias for existing enum from Alamofire?